tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/service/BigQueryService.java

```diff
@@ -59,4 +76,29 @@ public BigQueryException catchExceptionFromDryRun(String query) {
 
     return null;
   }
+
+  /**
+   * Fetch the names of all table from the BigQuery given the project and dataset. If the project
+   * and/or dataset do not exist or not visible to the provided service account, {@link
+   * com.google.cloud.bigquery.BigQueryException} will be thrown.
+   *
+   * <p>It is possible that a dataset contains a large amount of tables, so this method will only
+   * fetch maximum 1000 of them.
+   *
+   * @param projectId project id
```

Comment:
I don't the "param" comment make many sense so far(they are mostly self-explanary), as seen from other methods. Later you can just keep the comment of the method but leave out the comments for the params.

Reply:
Sounds good!